docs/ONNX_Runtime_for_Mobile_Platforms.md

```diff
@@ -20,21 +20,34 @@ Once you have cloned the repository, perform the following steps to create a min
 ### 1. Create ORT format model and configuration file with required operators
 
 We will use a helper python script to convert ONNX format models into ORT format models, and to create the configuration file for use with the minimal build.
-This will require the standard ONNX Runtime python package to be installed.
-  - Install the ONNX Runtime python package from https://pypi.org/project/onnxruntime/. Version 1.5.2 or later is required.
-    - `pip install onnxruntime`
-    - ensure that any existing ONNX Runtime python package was uninstalled first, or use `-U` with the above command to upgrade an existing package
-  - Copy all the ONNX models you wish to convert and use with the minimal build into a directory
-  - Convert the ONNX models to ORT format
-    - `python <ONNX Runtime repository root>/tools/python/convert_onnx_models_to_ort.py <path to directory containing one or more .onnx models>`
-      - For each ONNX model an ORT format model will be created with '.ort' as the file extension.
-      - A `required_operators.config` configuration file will also be created.
+
+The configuration file specifies what operator kernels to include in the build.
+This allows unused operator kernels to be pruned in order to decrease the binary size.
+
+It is also possible (and optional) to further prune the operator kernel implementations based on their input and output type usage detected in the ORT format models.
+This pruning is referred to as "operator type reduction" in this documentation.
+
+- The helper python script requires the standard ONNX Runtime python package to be installed. Install the ONNX Runtime python package from https://pypi.org/project/onnxruntime/. Version 1.5.2 or later is required.
+  To enable operator type reduction, version 1.7 or later is required.
+  - `pip install onnxruntime`
+  - Ensure that any existing ONNX Runtime python package was uninstalled first, or use `-U` with the above command to upgrade an existing package.
+
+- Additionally, if you want to enable operator type reduction, the Flatbuffers python package should be installed.
+  - `pip install flatbuffers`
+
+- Copy all the ONNX models you wish to convert and use with the minimal build into a directory.
+
+- Convert the ONNX models to ORT format
+  - `python <ONNX Runtime repository root>/tools/python/convert_onnx_models_to_ort.py <path to directory containing one or more .onnx models>`
+    - To enable operator type reduction, specify the `--enable_type_reduction` option.
+  - For each ONNX model an ORT format model will be created with '.ort' as the file extension.
+  - A `required_operators.config` configuration file will also be created.
 
```

Comment:
nit: if type reduction is enabled the default is a file called required_operators_and_types.config.